test/test_distributions.py

```diff
@@ -366,6 +366,21 @@ def get_sp_dist(jax_dist):
         np.broadcast_to(np.identity(3), (2, 3, 3)),
         None,
     ),
+    T(
+        dist.CAR,
+        0.0,
+        0.0,
+        0.1,
+        np.array(
+            [
+                [0.0, 1.0, 1.0, 0.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0, 0.0],
+            ]
+        ),
+        False,
+    ),
```

Comment:
+ a batch (with shape `(2,)`) of alpha
```
    T(
        dist.CAR,
        np.array([0., 1., 3., 4.]),
        np.array([-0.2, 0.3]),
        0.1,
        np.array(
            [
                [0.0, 1.0, 1.0, 0.0],
                [1.0, 0.0, 0.0, 1.0],
                [1.0, 0.0, 0.0, 1.0],
                [0.0, 1.0, 1.0, 0.0],
            ]
        ),
    ),
```
+ a batch (with shape `(2,)`) of tau
```
    T(
        dist.CAR,
        np.array([0., 1., 3., 4.]),
        np.array(0.1),
        np.array([0.3, 0.7]),
        np.array(
            [
                [0.0, 1.0, 1.0, 0.0],
                [1.0, 0.0, 0.0, 1.0],
                [1.0, 0.0, 0.0, 1.0],
                [0.0, 1.0, 1.0, 0.0],
            ]
        ),
    ),
```
+ a batch of W
```
    T(
        dist.CAR,
        np.array([0., 1., 3., 4.]),
        np.array(0.1),
        np.array([0.3, 0.7]),
        np.broadcast_to(np.array(
            [
                [0.0, 1.0, 1.0, 0.0],
                [1.0, 0.0, 0.0, 1.0],
                [1.0, 0.0, 0.0, 1.0],
                [0.0, 1.0, 1.0, 0.0],
            ]
        ), (2, 4, 4)),
    ),
```
+ sparse test
```
        partial(dist.CAR, is_sparse=True),
        np.array([0., 1., 3., 4.]),
        0.0,
        0.1,
        np.array(
            [
                [0.0, 1.0, 1.0, 0.0],
                [1.0, 0.0, 0.0, 1.0],
                [1.0, 0.0, 0.0, 1.0],
                [0.0, 1.0, 1.0, 0.0],
            ]
        ),
```